netci.groovy

```diff
@@ -1189,7 +1239,13 @@ combinedScenarios.each { scenario ->
                                         // Set time out
                                         setTestJobTimeOut(newJob, scenario)
                                         // Basic archiving of the build
-                                        Utilities.addArchival(newJob, "bin/Product/**,bin/obj/*/tests/**/*.dylib,bin/obj/*/tests/**/*.so")
+                                        if (scenario == 'coverage')
+                                        {
+                                            Utilities.addArchival(newJob, "bin/Product/**,bin/obj/**")
```

Comment:
What does bin/obj have in it?  IIRC this directory is pretty large.  Can you change this to be more specific to the required coverage data?